clang/test/Driver/sycl-offload-static-lib.cpp

```diff
@@ -4,6 +4,14 @@
 // REQUIRES: clang-driver
 // REQUIRES: x86-registered-target
 
+/// test behaviors of passing a fat static lib
+// Build a fat static lib that will be used for all tests
+// RUN: echo "void foo(void) {}" > %t1.cpp
+// RUN: %clangxx -target x86_64-unknown-linux-gnu -fsycl %t1.cpp -c -o %t1_bundle.o
+// RUN: llvm-ar cr %t.a %t1_bundle.o
+// RUN: llvm-ar cr %t.lo %t1_bundle.o
+// RUN: llvm-ar cr %t_2.a %t1_bundle.o
```

Comment:
I'm a bit confused - the archive appears unused, as no other additions were made to the test (?)

Reply:
The named archives are used in the test itself, but due to the additional verification that occurs with the change the check for the embedded library within the @<filelist> was failing.  So a true archive needs to be created.

Reply:
Cleaned out the .lo addition that isn't used, but the other files are referenced in the test.

Reply:
It's just that I'm failing to find any references to `%t1_bundle.o` in `test/Driver/sycl-offload-static-lib.cpp`. Could you please specify the using cases?

Reply:
Oh, there are no uses of `%t1_bundle.o`.  The uses are the generated archives.  The object is intermediate.

Reply:
Oh, that was rather blind of me...